Felienne/hedy
# Decreased size of achievements logo

**Description**
In this PR we decrease the size of the achievements logo and center the image to improve the UI. This change make the achievements page look less "crowded".

**Fix for**
This PR fixes #1664.

**How to test**
Log in and navigate to the achievements page. Notice that the logo within each achievement container is slightly smaller.

![Schermafbeelding 2022-01-12 om 15 40 57](https://user-images.githubusercontent.com/38155424/149161424-4d9d6057-0954-4487-b7b2-70f1b956d313.png)

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
